CHANGES.md

```diff
@@ -1,3 +1,7 @@
+### 0.6.0 (2020-04-03)
```

Comment:
should the date be today?  (2020-04-23)

Reply:
This is why we have pull requests for these things!